### ISSUE TITLE
Telekinesis can grab paper through walls

### ISSUE THREAD

**Issue by RazrPredator** [Skyrat-SS13/Skyrat-tg#2036]
How I came across the glitch in Round ID 116 on 12/3/2020

I had the job as a geneticist and after all the work was done I was wandering around. When the shuttle had been called I was near the bridge. I wanted to test if I could fold papers with the telekinesis gene and so I pressed ALT+Left click on a paper that was in bridge. Instead of the paper turning into an airplane as expected. The paper instead turned into an airplane and was immediately teleported into my hand. I swiftly returned the paper to the bridge.

Why is this important? 

Well, I discover it and was shocked that it happened. However, if this is a known glitch eventually then what is stopping people from stealing secured documents from areas they can't reach? This could lead to issues, besides it being LRP. Plus it's unfair since it offers the ability to take a paper from a room that you don't have access to.

Thank you for your time!

TLDR Steps to recreate.
-Get Telekineses
-Have a paper on the other side of a wall, which you can see through
-ALT+L Click the paper
-Paper should be teleported to your hand.
